src/utils.js

```diff
@@ -18,8 +19,32 @@ function getPulumiStack() {
  * @param {string?} stack Pulumi stack
  * @return {{}} Environment key-value pairs
  */
-function getEnvironmentVariables(twilioClient) {
+function getEnvironmentVariables(twilioClient, args) {
```

Comment:
If you change the definition of a function, can you also update the documentation (i.e. the comment above)? BTW, that documentation was wrong in the first place: i forgot to update it when I cleaned it up (that's what code review should be for 😛 )